app/assets/stylesheets/pages/home/_home-videos.scss

```diff
@@ -17,23 +17,19 @@ section.home-videos {
     }
 
     .home-videos-title {
-      h2 {
-        text-align: center;
-      }
+        h2 {
+            text-align: center;
+        }
     }
 
     .home-videos-player-wrapper {
-      position: relative;
-      padding-top: 40%;
-
-      @media only screen and (max-width: 768px) {
-        padding-top: 60%
-      }
+        position: relative;
+        padding-top: 59%;
```

Comment:
Isn't this too big in desktop?

Reply:
This is how it shows up to me:
https://d.pr/i/UFpP2S